index.html

```diff
@@ -9,27 +9,27 @@
   <link rel="stylesheet" media="(min-width:451px)" href="tablet.css">
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
 
 <body>
   <div class="section-1">
-    <header>
-      <div class="logo">
-        <a href="#" class="logo">My Logo</a>
-      </div>
-      <nav>
-        <div class="icon-div desktop-hidden">
-          <a href="javascript:void(0);" class="icon desktop-hidden">
-            <i class="fa fa-bars"></i>
-          </a>
+    <header class="menu-wrap">
+      <nav class="logo">
+        <a href="#" class="brand">David</a>
+        <input type="checkbox" class="toggler" id="toggle" onclick="toggle()"/>
+        <div class="hamburger">
+          <div></div>
         </div>
-        <div class="nav-links mobile-hidden">
-          <ol>
-            <li><a href="#" class="nav-link">Portfolio</a></li>
-            <li><a href="#" class="nav-link">About</a></li>
-            <li><a href="#" class="nav-link">Contact</a></li>
-          </ol>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li  onclick="remove()"><a href="#works" class="nav-links">Portfolio</a></li>
+                <li  onclick="remove()"><a href="#about" class="nav-links">About</a></li>
+                <li  onclick="remove()"><a href="#contact" class="nav-links">Contact</a></li>
+              </ul>
+            </div>
+          </div>
         </div>
       </nav>
```

Comment:
- You have done well so far, but your navbar still has some little differences from what is in the design template. Please note that when the mobile menu appears the background gets blurred a little bit. Please fix this 🙏 

Your project:
<img width="310" alt="wddcd" src="https://user-images.githubusercontent.com/73607512/217256593-1543f148-53d8-416f-9ba9-333dd3186927.PNG">

Design template:
<img width="144" alt="nedtaethe" src="https://user-images.githubusercontent.com/73607512/217256668-a9d8ee38-6843-4317-84ce-7e23744589d3.PNG">